app/jobs/further_education_payments/provider_overdue_chaser_job.rb

```diff
@@ -0,0 +1,70 @@
+module FurtherEducationPayments
+  class ProviderOverdueChaserJob < ApplicationJob
+    queue_as :user_data
+
+    # Temporal safeguard: don't send emails for claims older than this
+    CUTOFF_PERIOD = 6.months
+
+    def perform
+      # Find all unverified FE eligibilities with assigned providers
+      unverified_eligibility_ids = Policies::FurtherEducationPayments::Eligibility
+        .where(provider_verification_completed_at: nil)
+        .where.not(provider_assigned_to_id: nil)
```

Comment:
are we sure we only want to deal with claims that are assigned? all the unassigned claims will not be included in these emails, is that desired?